apps/backend/test/constants/groups-test.constant.ts

```diff
@@ -2,15 +2,18 @@ import {CreateGroupDto} from '../../src/groups/dto/create-group.dto';
 
 export const GROUP_1 = {
   name: 'Heimdall Group',
-  public: true
+  public: true,
+  desc: ''
 };
 
 export const PRIVATE_GROUP = {
   name: 'Private Heimdall Group',
-  public: false
+  public: false,
+  desc: ''
```

Comment:
we should also have a group with an actual description.

Reply:
Sure, I'll add a description to PRIVATE_GROUP